### PR TITLE
Remove unsupported fields from PutMappingRequest (Issue # 62)

### PR DESCRIPTION
**Pull request by Daniel-Houston**
Signed-off-by: Daniel Houston <dmhous@amazon.com>

### Description
Remove unsupported fields from PutMappingRequest.

PutMappingRequest was serializing fields into the json body which were not supported by the API. This was causing a response error from opensearch

### Issues Resolved
Closes [Issue-62](https://github.com/opensearch-project/opensearch-java/pull/63)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dblock** [comment]
@Daniel-Houston needs a line in https://github.com/opensearch-project/opensearch-java/blob/main/CHANGELOG.md please and you're all set!

**Comment by Daniel-Houston** [comment]
I'll get to this just trying to finish up a few other things

File: java-client/src/main/java/org/opensearch/client/opensearch/indices/PutMappingRequest.java [hunk 1]
```diff
@@ -293,26 +170,6 @@ public final Time masterTimeout() {
 		return this.masterTimeout;
 	}
 
-	/**
-	 * Specify timeout for connection to cluster-manager
-	 * <p>
-	 * API name: {@code cluster_manager_timeout}
-	 */
-	@Nullable
-	public final Time clusterManagerTimeout() {
```
Comment (VachaShah) [inline java-client/src/main/java/org/opensearch/client/opensearch/indices/PutMappingRequest.java:302]: ClusterManagerTimeout is part of the inclusive naming #166, so we can keep that.

**Comment by VachaShah** [comment]
@Daniel-Houston want to finish this?

**Comment by VachaShah** [comment]
Finishing up this PR in #597. Thank you @Daniel-Houston for the changes!

**Comment by VachaShah** [comment]
Done via #597

**Pull request closed without merging**